src/controllers/v1/comments.js

```diff
@@ -46,7 +46,8 @@ module.exports = class comments {
 				req.params.id ? req.params.id : '',
 				req.query.resource_id,
 				req.body,
-				req.decodedToken.id
+				req.decodedToken.id,
+				req.decodedToken.organization_id
```

Comment:
@priyanka-TL do we not need to pick from resource the org id?

Reply:
it is passed for create activity, but its not required now because the review status will be inprogress when click on start button